sunscreen_compiler_common/src/context.rs

```diff
@@ -444,5 +463,48 @@ where
      */
     pub fn add_edge(&mut self, from: NodeIndex, to: NodeIndex, edge: EdgeInfo) {
         self.graph.add_edge(from, to, edge);
+
+        #[cfg(feature = "debugger")]
```

Comment:
Maybe do this lazily in the REST handler?

Reply:
I won't die on this hill, but we should avoid storing redundant data when we can avoid it.